packages/editor/src/components/provider/index.js

```diff
@@ -215,18 +195,13 @@ export default compose( [
 			__experimentalGetReusableBlocks,
 		} = select( 'core/editor' );
 		const { canUser } = select( 'core' );
-		const { getInstalledBlockTypes } = select( 'core/block-directory' );
-		const { getBlocks } = select( 'core/block-editor' );
-
-		const downloadableBlocksToUninstall = differenceBy( getInstalledBlockTypes(), getBlocks(), 'name' );
```

Comment:
Can someone confirm how it works? I read it as:
> when all instances of the installed block get removed from the content then the block gets automatically removed as well

Reply:
cc @talldan @noisysocks

Reply:
Realised I had unsubmitted pending review comments, so that's why my comment is before the ping ⌛️